ZenPacks/zenoss/ZenPackLib/lib/base/ZenPack.py

```diff
@@ -290,13 +290,19 @@ def template_changed(self, app, existing, new_spec):
 
     def create_device_class(self, app, dcspec):
         ''''''
+        exists = False
         try:
             dcObject = app.dmd.Devices.getOrganizer(dcspec.path)
+            exists = True
         except KeyError:
             self.LOG.info('Creating DeviceClass {}'.format(dcspec.path))
             dcObject = app.dmd.Devices.createOrganizer(dcspec.path)
 
         for zprop, value in dcspec.zProperties.iteritems():
+            # Avoid setting zProperties on an existing device class
+            if exists:
+                self.LOG.debug('Not setting "{}" to "{}" on existing device class ({})'.format(zprop, value, dcspec.path))
```

Comment:
This log should probably only be produced if the YAML has a different value than the device class.

Also, you may want to consider using the `!r` formatted rather than creating your own quotes.

```python
"Not setting {} to {!r} on {}".format(zprop, value, dcspec.path)
```